packages/@vuepress/shared-utils/src/inferTitle.ts

```diff
@@ -9,11 +9,10 @@ import deeplyParseHeaders from './deeplyParseHeaders'
  */
 
 export = function (frontmatter: Record<string, any>, strippedContent: string): string | void {
-  if (frontmatter.home) {
-    return 'Home'
-  }
   if (frontmatter.title) {
     return deeplyParseHeaders(frontmatter.title)
+  } else if (frontmatter.home) {
```

Comment:
the `else` here is redundant

Reply:
Yeah, my bad! 🙈
How can I fix it? Should I make another PR?

Reply:
Just push to the same branch, and current PR will update